calico-policies/public-network-isolation/au-syd/allow-ibm-ports-public.yaml

```diff
@@ -63,6 +63,13 @@ spec:
       nets:
       - 172.20.0.0/24
     protocol: TCP
+  - action: Allow
+    destination:
+      nets:
+      # Allows communication from host to a cluster IP service.  This is need in these public
+      # policies for now due to https://github.com/projectcalico/felix/pull/2582
+      # If you specified a custom pod subnet when you created the cluster, use that CIDR instead.
```

Comment:
```suggestion
      # Allows communication from a worker node to a ClusterIP service, which is required
      # due to https://github.com/projectcalico/felix/pull/2582. If you specified a custom pod subnet
      # when you created a classic cluster, or if you use a VPC cluster, use that CIDR instead.
```

Reply:
and same ^ throughout

Reply:
I added this to ca-tor, and made the suggested modifications, except I did keep the reference to "these public policies" since this rule is already in the private policies and makes sense there.  It is needed in these public policies (where it doesn't really make sense) due to the bug I reference